src/main.rs

```diff
@@ -26,13 +35,13 @@ fn cli() -> Command {
                         .arg(
                             arg!(--"start-block" <START_BLOCK>)
                                 .help("Ethereum block number to start state import from")
-                                .default_value(ethereum::GENESIS_BLOCK.to_string())
+                                .default_value(GENESIS_BLOCK.to_string())
```

Comment:
No strong opinion on this one, but just explaining my thinking here: I specifically kept the `ethereum` module prefix to denote what genesis block was in question. I'd expect the L2 to have their own genesis block as well, although currently it's not involved here. i.e. code readability aspect :slightly_smiling_face:

Reply:
Gotcha, yeah, I wasn't quite sure why the need for that prefix existed but this makes a lot of sense.